errors.go

```diff
@@ -67,6 +67,10 @@ var (
 
 	// ErrUnknownError is returned when notmuch returns NULL indicating an error.
 	ErrUnknownError = errors.New("unknown error occured")
+
+	// ErrNoReplies is returned if a message has no replies or if the message's C
+	// pointer did not come from a thread.
+	ErrNoReplies = errors.New("message has no replies")
```

Comment:
I renamed it to `ErrNoRepliesOrPointerNotFromThread` a mouthful but I couldn't think of a different/shorter name.